### PR TITLE
chore: Update VS Code Extension Link

### PR DESCRIPTION
**Pull request by Savio-Sou**
# Description

## Summary\*

This PR updates the docs to point readers to the new [Noir Language Support VS Code Extension](https://marketplace.visualstudio.com/items?itemName=noir-lang.vscode-noir), which exposes Language Server Protocol features in the IDE when paired with ≥v0.7.1 Nargo installed.

# PR Checklist\*

- [x] I have tested the changes locally.
- [x] I have formatted the changes with [Prettier](https://prettier.io/) and/or `cargo fmt` on default settings.

### CODE REVIEW

**Review by phated — APPROVED** [review]
LGTM

**Comment by Savio-Sou** [comment]
Thanks Blaine!

(For whomever reading, this PR include other changes proposed in https://github.com/noir-lang/docs/pull/212. For best practices we should review and merge that first before this.)

**Comment by Savio-Sou** [comment]
(To readers from the future: This PR includes other changes proposed in https://github.com/noir-lang/docs/pull/212. For best practices we should review and merge that first before merging this.)

**Pull request merged**
